### PR TITLE
add possibility to disable the multiselect dropdown

### PR DESCRIPTION
**Pull request by MichelML**

### CODE REVIEW

File: src/components/select/MultiSelectConnected.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ const parentDropTarget = {
 @DropTarget(DraggableSelectedOptionType, parentDropTarget, (connect: any) => ({
     connectDropTarget: connect.dropTarget(),
 }))
-export class MultiSelectConnected extends React.Component<IMultiSelectProps, {}> {
+export class MultiSelectConnected extends React.Component<IMultiSelectProps & React.ButtonHTMLAttributes<HTMLButtonElement>> {
```
Comment (pochretien) [inline src/components/select/MultiSelectConnected.tsx]: Nice call. to help with other project who want to use the IMultiSelectProps interface, we need to do this
`interface IMultiSelectProps extends React.ButtonHTMLAttributes<HTMLButtonElement>`

Comment (MichelML) [reply src/components/select/MultiSelectConnected.tsx]: changed, thanks

File: src/components/select/MultiSelectConnected.tsx [hunk 2]
```diff
@@ -19,7 +19,10 @@ import {Tooltip} from '../tooltip/Tooltip';
 import {ISelectButtonProps, ISelectProps, SelectConnected} from './SelectConnected';
 import {SelectSelector} from './SelectSelector';
 
-export interface IMultiSelectOwnProps extends ISelectProps, IDropTargetProps {
+export interface IMultiSelectOwnProps extends
```
Comment (pochretien) [inline src/components/select/MultiSelectConnected.tsx]: GG

File: src/components/select/SelectConnected.tsx [hunk 3]
```diff
@@ -21,7 +21,7 @@ export interface ISelectSpecificProps {
 }
 
 export interface ISelectOwnProps {
-    id: string;
+    id?: string;
```
Comment (GermainBergeron) [inline src/components/select/SelectConnected.tsx]: I'm pretty sure the id is required

Comment (MichelML) [reply src/components/select/SelectConnected.tsx]: changed

**Pull request merged**